src/Apps/Artwork/Utils/createCollectUrl.tsx

```diff
@@ -47,6 +47,7 @@ const filterCategories = {
   Architecture: "architecture",
   "Books and Portfolios": "books-and-portfolios",
   "Design/Decorative Art": "design",
+  "Digital Art": "digital-art",
```

Comment:
**thought (non-blocking):** Not totally sure how to verify this, since the price-in-context feature appears to be wonky.